src/style.css

```diff
@@ -1,3 +1,39 @@
 body {
   margin: 0;
 }
+
+@font-face {
+  font-family: Roboto;
+  src: url(https://fonts.google.com/specimen/Roboto);
+}
```

Comment:
The `src` attribute in the `@font-face` rule should point to a valid font file URL, such as a `.woff`, `.woff2`, or similar file type, rather than a Google Fonts page URL. Consider using the correct URL format provided by Google Fonts for embedding fonts.